Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -1904,6 +1904,106 @@ theorem lt_iff_sub_pos {x y : PGame} : x < y ↔ 0 < y - x :=
       ⟩
 #align pgame.lt_iff_sub_pos SetTheory.PGame.lt_iff_sub_pos
 
+/-! ### Inserting an option -/
+
+/-- The pregame constructed by inserting x' as a new left option into x. -/
+def insertLeft (x : PGame) (x' : PGame) : PGame :=
+  match x with
+  | mk xl xr xL xR => mk (Sum xl PUnit) xr (fun i' =>
+      match i' with
+      | .inl i => xL i
+      | .inr () => x'
+    ) xR
+
+/-- A new left option cannot hurt Left. -/
+lemma insert_left (x : PGame) (x' : PGame) : x ≤ insertLeft x x' := by
+  rw [le_def]
+  constructor
+  · intro i
+    left
+    rcases x with ⟨xl, xr, xL, xR⟩
+    simp only [insertLeft, leftMoves_mk, moveLeft_mk, Sum.exists, exists_const]
+    left
+    use i
+  · intro j
+    right
+    rcases x with ⟨xl, xr, xL, xR⟩
+    simp only [rightMoves_mk, moveRight_mk, insertLeft]
+    use j
+
+/-- Adding a gift horse left option does not change the value of x. -/
```

Comment:
Can you explain "gift horse" in this docstring, since it has not been mentioned before?